src/defaults/html.pipe.js

```diff
@@ -59,6 +61,8 @@ const htmlpipe = (cont, context, action) => {
     .every(dump.record)
     .every(validate).when((ctx) => !production() && !ctx.error)
     .every(timer.update)
+    .use(fetchFstab).expose('fstab')
```

Comment:
From what I remember, @trieloff wanted to fetch in parallel instead of in sequence to reduce execution time.
So not sure if that has changed now, but I'll need to align #516 according to the consensus

Reply:
this step only schedules the donwload but doesn't wait for it....